src/RedHat.AspNetCore.Server.Kestrel.Transport.Linux/TransportThread.TSocket.cs

```diff
@@ -61,8 +59,13 @@ public struct ReceiveMemoryAllocation
             private const int EventControlRegistered = (int)SocketFlags.EventControlRegistered;
             public const int EventControlPending = (int)SocketFlags.EventControlPending;
 
-            private static readonly int MaxBufferSize = KestrelMemoryPool.MinimumSegmentSize;
-            private static readonly int BufferMargin = MaxBufferSize / 4;
+            // Copied from LibuvTransportOptions.MaxReadBufferSize
+            private static readonly int PauseInputWriterThreashold = 1024 * 1024;
```

Comment:
nit: `PauseInputWriterThreashold` -> `PauseInputWriterThreshold` (same for `PauseOutputWriterThreashold`)